assets/js/mixins/formatter.js

```diff
@@ -335,5 +391,63 @@ export default {
       // TODO: handle fahrenheit
       return this.fmtNumber(value, 1, "celsius");
     },
+    getWeekdaysList: function (weekdayFormat) {
+      const { format } = new Intl.DateTimeFormat(this.$i18n?.locale, { weekday: weekdayFormat });
+      const mondayToSaturday = [7, 8, 9, 10, 11, 12].map((day, index) => {
+        return { name: format(new Date(Date.UTC(2021, 5, day))), value: index + 1 };
+      });
+      const sunday = { name: format(new Date(Date.UTC(2021, 5, 6))), value: 0 };
+      return [...mondayToSaturday, sunday];
+    },
+    getShortenedWeekdaysLabel: function (selectedWeekdays) {
+      if (0 === selectedWeekdays.length) {
```

Comment:
style

```suggestion
      if (selectedWeekdays.length === 0) {
```

Du verwendest das an mehreren Stellen so. Ich weiß, dass diese schreibweise ab und an als good practice beschrieben wird um versehentliche Zuweisungen zu vermeiden. Ich find das aber schwerer lesbar. So herum würde man es in menschlicher Sprache auch nie sagen. Daher lieber erst der Wert und dann die Erwartung.